test/unit/js/infiniteGrid.test.js

```diff
@@ -90,8 +90,8 @@ test("release event handler after destroy", function(assert) {
 
 	// Then
 	setTimeout(function() {
-		equal(scrollCount, 1, "should exist scroll event");
-		equal(resizeCount, 1, "should exist resize event");
+		ok(scrollCount > 0, "should exist scroll event");
```

Comment:
It would be better if it has some comments.

// resize event is fired twice in IE8

How about you?